packages/lab/src/Table/stories/TableColumnRenderers.stories.js

```diff
@@ -0,0 +1,938 @@
+/* eslint-disable storybook/default-exports, no-nested-ternary */
+
+// build warning is expected:
+// Skipping packages/lab/src/Table/stories/TableColumnRenderers.stories.js: NoMetaError: CSF: missing default export
+// this .stories.js file is only intended to be parsed for retrieving the stories' source code,
+// but they are rendered by the TableHooks.stories.mdx file
+
+import React, { useMemo, useState } from "react";
+import { Ban } from "@hitachivantara/uikit-react-icons";
+
+import { HvEmptyState, HvPagination, HvTypography } from "@hitachivantara/uikit-react-core";
+
+import {
+  HvTable,
+  HvTableBody,
+  HvTableCell,
+  HvTableContainer,
+  HvTableHead,
+  HvTableHeader,
+  HvTableRow,
+  hvTextColumn,
+  hvDateColumn,
+  hvExpandColumn,
+  hvSwitchColumn,
+  hvNumberColumn,
+  hvTagColumn,
+  hvDropdownColumn,
+  hvProgressColumn,
+  useHvTable,
+  useHvPagination,
+  useHvRowExpand,
+} from "../..";
+
+import { makeRenderersData } from "./utils";
+
+export const AllColumnRenderers = () => {
+  const getColumns = () => [
+    hvSwitchColumn(
+      {
+        Header: "isDisabled",
+        accessor: "isDisabled",
+        style: { minWidth: 130 },
+      },
+      "default",
+      "yes",
+      "no",
+      {
+        disabled: true,
+      }
+    ),
+    hvExpandColumn(
+      { Header: "Title", accessor: "name", style: { maxWidth: 100 } },
+      "expand",
+      "collapse",
+      () => true
+    ),
+    hvDateColumn({ Header: "Time", accessor: "createdDate", style: { minWidth: 50 } }),
```

Comment:
this could format the date in some shape or form instead display the raw date object.
![Screenshot 2022-01-24 at 16 32 11](https://user-images.githubusercontent.com/1148693/150824040-51e47fa4-95ae-4d7c-afde-c36e6446e0a0.png)